translate/rus.json

```diff
@@ -296,7 +299,7 @@
 "panel options":"настройки панели",
 "panel":"панель",
 "parameters":"параметры",
-"percent width":"ширина процента",
+"percent width":"размер процентов",
```

Comment:
Originally it would have had slightly longer translation, but it was shortened for GUI appearance purposes. The problem with this translation is the fact that it covers one 'percentage' instead of multiple. I fixed it by putting the word 'процент' in it's plural form. The case of the noun remained the same.